custom_components/smartthinq_sensors/wideq/devices/ac.py

```diff
@@ -684,6 +694,27 @@ def is_mode_jet_available(self):
             return True
         return False
 
+    @property
+    def is_pm10_supported(self):
+        """Return if PM sensors are supported."""
+        if self._is_pm_supported is None:
+            self._is_pm_supported = self._is_mode_supported(SUPPORT_PM)
+        return self._is_pm_supported[0]
+
+    @property
+    def is_pm25_supported(self):
+        """Return if PM sensors are supported."""
+        if self._is_pm_supported is None:
+            self._is_pm_supported = self._is_mode_supported(SUPPORT_PM)
+        return self._is_pm_supported[2]
+
+    @property
+    def is_pm1_supported(self):
+        """Return if PM sensors are supported."""
+        if self._is_pm_supported is None:
+            self._is_pm_supported = self._is_mode_supported(SUPPORT_PM)
+        return self._is_pm_supported[1]
+
```

Comment:
I'm not sure that I like to have 3 different properties, because I'm planning to replace cache attributes with  [cached_property](https://docs.python.org/3/library/functools.html#functools.cached_property) and I couldn't use in this case.
Apart this I like this approach, I'll think about this later.
Maybe should be better to order the array in different way (PM1, PM25 and PM10)? Or you use this order for a specific reason (I'm not a PM expert)?

Reply:
Because from a human readability standpoint I liked:
1 = PM1
2 = PM2
0 = PM10 ;-)